src/calculateRentalCost.js

```diff
@@ -9,26 +8,42 @@
  * You will need a rental car in order for you to get around in your vacation.
  * The manager of the car rental makes you some good offers.
  *
- * Every day you rent the car costs $40.
- * If you rent the car for 7 or more days, you get $50 off your total.
- * Alternatively, if you rent the car for 3 or more days,
- * you get $20 off your total.
+ * Every day you rent the car costs $dailyCost.
+ * Alternatively, if you rent the car for bigWeekend or more days,
+ * you get $minSale off your total.
  *
  * Implement calculateRentalCost function
  * which returns the total amount for different count of days.
  *
  * Examples:
- *  - calculateRentalCost(1) === 40
- *  - calculateRentalCost(3) === 100
- *  - calculateRentalCost(7) === 230
+ *  - calculateRentalCost(1) === dailyCost
+ *  - calculateRentalCost(bigWeekend) === 100
+ *  - calculateRentalCost(weekend) === 2bigWeekend0
  *
  * @param {number} days
  *
  * @return {number}
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const dailyCost = 40;
+  const minSale = 20;
+  const maxSale = 50;
+  const bigWeekend = 3;
+  const weekend = 7;
+
+  if (days >= bigWeekend && days < weekend) {
+    return days * dailyCost - minSale;
+  }
 
+  if (days >= weekend) {
+    return days * dailyCost - maxSale;
+  }
+
+  if (days < bigWeekend && days > 0) {
+    return days * dailyCost;
+  } else {
+    return 0;
+  }
```

Comment:
```suggestion
  if (days <= 3) {
    return days * dailyCost - minSale;
  }

  if (days <= 7) {
    return days * dailyCost - maxSale;
  }
  
  return 
```
 two ifs and return will be enough